test/dotcom_web/live/trip_planner_test.exs

```diff
@@ -3,9 +3,39 @@ defmodule DotcomWeb.Live.TripPlannerTest do
 
   import Mox
   import Phoenix.LiveViewTest
+  import Test.Support.Factories.TripPlanner.TripPlanner, only: [limit_route_types: 1]
+
+  alias OpenTripPlannerClient.Test.Support.Factory, as: OtpFactory
 
   setup :verify_on_exit!
 
+  def stub_otp_results(itineraries) do
+    expect(OpenTripPlannerClient.Mock, :plan, fn _ ->
+      {:ok, %OpenTripPlannerClient.Plan{itineraries: itineraries}}
+    end)
+  end
+
+  def stub_populated_otp_results() do
+    # Uhhh the OTP factory will generate with any route_type value but our
+    # parsing will break with unexpected route types
+    itineraries =
+      OtpFactory.build_list(3, :itinerary)
+      |> Enum.map(&limit_route_types/1)
+
+    stub_otp_results(itineraries)
```

Comment:
I don't think we do this in a factory (I didn't add this - I just moved it), and I do think we should. I was thinking that fixing that could be a follow-up PR, so that the depart-at buttons can still go-live.

Reply:
That's fine as long as we do it later. I know the factory does this, but it also parses it. Might be worth trying to use the factory directly: https://github.com/mbta/dotcom/blob/8266c5671ff3af8454c1de85527883f916533c31/test/support/factories/trip_planner/trip_planner.ex#L10